### ISSUE TITLE
Order the main dry box

### ISSUE THREAD

**Issue by anardek** [NCSU-CHAZ/BYO-Hydrographic-RV#2]

**Comment by anardek** [comment]
For the laptop, we will also need:

- [ ] touch screen interface (10 inch display)
- [ ] foldable waterproof keyboard
- [ ] 3 underwater cables for computer and power box (VGA cable to the display, power and connection cable for the touchscreen); Vartek (?) can do the mating for a small fee -- my notes were bad on this. Maybe we will know more looking at the photos Brett sent.

**Comment by hkmccraw** [comment]
@anardek @bwebb-southal here is some info I have compiled on cappuccino PCs. Our options are dependent on how many LAN ports we need--both options below have two. Both are the same size but the more expensive option has two removable SD card slots as opposed to internal slots on the cheaper option.

Cappuccino PC:   

Cheaper Option: https://www.cappuccinopc.com/Mini-PC/echo-246f-Embedded-Fanless-Mini-PC-with-i7-CoffeeLake-and-Dual-NIC.asp 
More expensive option: https://www.cappuccinopc.com/Industrial-PC/Echo246F-PoE-fanless-mini-pc-with-4-PoE-Ports-and-3-SIM-Slots.asp  

![image](https://user-images.githubusercontent.com/105874556/182420857-79071382-f352-4710-9525-69af09b81faf.png)

**Comment by hkmccraw** [comment]
Here is a battery option for the dry box: [](https://dakotalithium.com/product/dakota-lithium-12v-12ah-18-amp-hour-lifepo4-iron-phosphate-battery/
)

**Comment by anardek** [comment]
Notes from call with Brett:
- need more than 1 battery wired in parallel (lithium); will just need to fit in the dry box [dakota lithium](https://dakotalithium.com/?gclid=CjwKCAjwkaSaBhA4EiwALBgQaIsxrLzdrJWuU7J1h1iJJ2skUkj-PgNHAsmkOKNar1IH61IOjcHouBoChakQAvD_BwE))
- for cappuccino pc, just make sure it is fanless, 4-6 usb inputs, that the VGA cable is compatible with the touchscreen (i.e., need to choose a touchscreen first, see the emailed quote from Bret)
- screen: can send the cables to vartech and have them splice for us -- this sounded complicated but less complicated than doing ourselves [Vartech screen](https://vartechsystems.com/?utm_term=vartech%20systems&utm_campaign=%2A%2ALP%20Branded&utm_source=adwords&utm_medium=ppc&hsa_acc=6642844538&hsa_cam=13981587584&hsa_grp=121903616061&hsa_ad=535150644157&hsa_src=g&hsa_tgt=kwd-296083805169&hsa_kw=vartech%20systems&hsa_mt=e&hsa_net=adwords&hsa_ver=3&gclid=CjwKCAjwkaSaBhA4EiwALBgQaA0tq-eJHTBiVIxQGWPTm2oTVtx4rCxppwAfXFaDVrBiCe8251r1wRoCkFEQAvD_BwE)
- look into buying the LinQ accessories directly and drilling them into the dry box
- for display cables: [subcon](https://www.macartney.com/what-we-offer/systems-and-products/connectors/subconn/\) or [impulse](https://impulsetechnologies.com/)

**Comment by anardek** [comment]
@bwebb-southal -- we measured out the space we will need in the dry box and the LinQ boxes we ordered won't work. We are trying to decide if we should get a waterproof pelican and add the LinQ clips to the bottom or just use the cooler we currently have with the clips (plenty of space but only water resistant; we will of course need to drill through it for the cords). What are your thoughts? I feel like attaching the LinQ connectors to the Pelican will probably make the Pelican less waterproof anyway....

**Comment by anardek** [comment]
> @anardek @bwebb-southal here is some info I have compiled on cappuccino PCs. Our options are dependent on how many LAN ports we need--both options below have two. Both are the same size but the more expensive option has two removable SD card slots as opposed to internal slots on the cheaper option.
> 
> Cappuccino PC:   
> 
> Cheaper Option: [https://www.cappuccinopc.com/Mini-PC/echo-246f-Embedded-Fanless-Mini-PC-with-i7-CoffeeLake-and-Dual-NIC.asp ](https://www.cappuccinopc.com/Mini-PC/echo-246f-Embedded-Fanless-Mini-PC-with-i7-CoffeeLake-and-Dual-NIC.asp%C2%A0) More expensive option: [https://www.cappuccinopc.com/Industrial-PC/Echo246F-PoE-fanless-mini-pc-with-4-PoE-Ports-and-3-SIM-Slots.asp  ](https://www.cappuccinopc.com/Industrial-PC/Echo246F-PoE-fanless-mini-pc-with-4-PoE-Ports-and-3-SIM-Slots.asp%C2%A0%C2%A0)
> 
> ![image](https://user-images.githubusercontent.com/105874556/182420857-79071382-f352-4710-9525-69af09b81faf.png)

Just a note for dummies: buy the option with the Windows OS (not Linux)

**Comment by anardek** [comment]
> Notes from call with Brett:
> 
> * need more than 1 battery wired in parallel (lithium); will just need to fit in the dry box [dakota lithium](https://dakotalithium.com/?gclid=CjwKCAjwkaSaBhA4EiwALBgQaIsxrLzdrJWuU7J1h1iJJ2skUkj-PgNHAsmkOKNar1IH61IOjcHouBoChakQAvD_BwE))
> * for cappuccino pc, just make sure it is fanless, 4-6 usb inputs, that the VGA cable is compatible with the touchscreen (i.e., need to choose a touchscreen first, see the emailed quote from Bret)
> * screen: can send the cables to vartech and have them splice for us -- this sounded complicated but less complicated than doing ourselves [Vartech screen](https://vartechsystems.com/?utm_term=vartech%20systems&utm_campaign=%2A%2ALP%20Branded&utm_source=adwords&utm_medium=ppc&hsa_acc=6642844538&hsa_cam=13981587584&hsa_grp=121903616061&hsa_ad=535150644157&hsa_src=g&hsa_tgt=kwd-296083805169&hsa_kw=vartech%20systems&hsa_mt=e&hsa_net=adwords&hsa_ver=3&gclid=CjwKCAjwkaSaBhA4EiwALBgQaA0tq-eJHTBiVIxQGWPTm2oTVtx4rCxppwAfXFaDVrBiCe8251r1wRoCkFEQAvD_BwE)
> * look into buying the LinQ accessories directly and drilling them into the dry box
> * for display cables: [subcon](https://www.macartney.com/what-we-offer/systems-and-products/connectors/subconn/) or [impulse](https://impulsetechnologies.com/)

I think my notes were wrong. Just one 18 Ah Lithium Ion battery.

**Comment by bwebb-southal** [comment]
> > @anardek @bwebb-southal here is some info I have compiled on cappuccino PCs. Our options are dependent on how many LAN ports we need--both options below have two. Both are the same size but the more expensive option has two removable SD card slots as opposed to internal slots on the cheaper option.
> > Cappuccino PC:   
> > Cheaper Option: [https://www.cappuccinopc.com/Mini-PC/echo-246f-Embedded-Fanless-Mini-PC-with-i7-CoffeeLake-and-Dual-NIC.asp ](https://www.cappuccinopc.com/Mini-PC/echo-246f-Embedded-Fanless-Mini-PC-with-i7-CoffeeLake-and-Dual-NIC.asp%C2%A0) More expensive option: [https://www.cappuccinopc.com/Industrial-PC/Echo246F-PoE-fanless-mini-pc-with-4-PoE-Ports-and-3-SIM-Slots.asp  ](https://www.cappuccinopc.com/Industrial-PC/Echo246F-PoE-fanless-mini-pc-with-4-PoE-Ports-and-3-SIM-Slots.asp%C2%A0%C2%A0)
> > ![image](https://user-images.githubusercontent.com/105874556/182420857-79071382-f352-4710-9525-69af09b81faf.png)
> 
> Just a note for dummies: buy the option with the Windows OS (not Linux)

Oh no! I actually made that mistake (but in reverse) on the purchase of a $35k supercomputer so you are in good company!

**Comment by bwebb-southal** [comment]
> > Notes from call with Brett:
> > 
> > * need more than 1 battery wired in parallel (lithium); will just need to fit in the dry box [dakota lithium](https://dakotalithium.com/?gclid=CjwKCAjwkaSaBhA4EiwALBgQaIsxrLzdrJWuU7J1h1iJJ2skUkj-PgNHAsmkOKNar1IH61IOjcHouBoChakQAvD_BwE))
> > * for cappuccino pc, just make sure it is fanless, 4-6 usb inputs, that the VGA cable is compatible with the touchscreen (i.e., need to choose a touchscreen first, see the emailed quote from Bret)
> > * screen: can send the cables to vartech and have them splice for us -- this sounded complicated but less complicated than doing ourselves [Vartech screen](https://vartechsystems.com/?utm_term=vartech%20systems&utm_campaign=%2A%2ALP%20Branded&utm_source=adwords&utm_medium=ppc&hsa_acc=6642844538&hsa_cam=13981587584&hsa_grp=121903616061&hsa_ad=535150644157&hsa_src=g&hsa_tgt=kwd-296083805169&hsa_kw=vartech%20systems&hsa_mt=e&hsa_net=adwords&hsa_ver=3&gclid=CjwKCAjwkaSaBhA4EiwALBgQaA0tq-eJHTBiVIxQGWPTm2oTVtx4rCxppwAfXFaDVrBiCe8251r1wRoCkFEQAvD_BwE)
> > * look into buying the LinQ accessories directly and drilling them into the dry box
> > * for display cables: [subcon](https://www.macartney.com/what-we-offer/systems-and-products/connectors/subconn/) or [impulse](https://impulsetechnologies.com/)
> 
> I think my notes were wrong. Just one 18 Ah Lithium Ion battery.

No, we do use two 18ah batteries wired in parallel to give us 12VDC and 36ah of capacity. Look at the amperage draw rate on the display and the computer. The display sucks down a lot of current. You might be able to get away with only one, but I’d suggest running the numbers to make sure.

**Comment by bwebb-southal** [comment]
Nope, nothing magical about this one. I believe we used a 4-pin SubConn connector as the USB touch interface was only using 4 wires. We cut the end off and marked the wires, then used a continuity checker to ensure that we had the correct wires connected to one another from end-to-end. We did not do anything fancy inside the box... we just used the end of the USB that we had snipped off and soldered that to the leads coming from the back of the bulkhead connector.
